test/fixtures.spec.js

```diff
@@ -16,13 +16,56 @@
  */
 
 const path = require('path');
-const {spawn} = require('child_process');
+const {spawn, execSync} = require('child_process');
 
-module.exports.describe = function({testRunner, expect, product, playwrightPath, FFOX, CHROMIUM, WEBKIT}) {
+module.exports.describe = function({testRunner, expect, product, playwright, playwrightPath, defaultBrowserOptions, WIN, FFOX, CHROMIUM, WEBKIT}) {
   const {describe, xdescribe, fdescribe} = testRunner;
   const {it, fit, xit, dit} = testRunner;
   const {beforeAll, beforeEach, afterAll, afterEach} = testRunner;
 
+  async function testSignal(action) {
+    const options = Object.assign({}, defaultBrowserOptions, {
```

Comment:
nit: If you want to be fancy, you can do
`{..defaultBrowserOptions, dumpio: false, webSocket: true}` instead of `Object.assign`. Up to you :)